testdrive/utils.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 const hashicorpReleasesURL = "https://releases.hashicorp.com"
-const terraformVersion = "1.3.5"
+const terraformVersion = "1.3.6" // renovate: datasource=github-releases depName=hashicorp/terraform versioning=hashicorp
```

Comment:
Did you mean to change 1.3.5 to 1.3.6? It's not changed in the other places.

Reply:
ah, [runatlantis:renovate/configure](https://github.com/runatlantis/atlantis/tree/renovate/configure)  branch contains 1.3.5, and main branch contains 1.3.6.

So could you update https://github.com/runatlantis/atlantis/pull/2681?

Reply:
Not sure what I'm updating in that pr ? Once this PR is merged, it will close #2681

Reply:
https://github.com/runatlantis/atlantis/pull/2681  PR is renovate introduce confirmation PR.

So it is better to do followings.

1. merge this #2807 into #2681
2. confirm renovate config is all valid on #2681 PR description which will be updated by renovate
3. merge #2681 into main

Reply:
or not need it, I will change this PR base branch to `main`.